components/molecule/tabs/src/index.scss

```diff
@@ -0,0 +1,168 @@
+@import '~@schibstedspain/sui-theme/lib/settings';
+@import '~@schibstedspain/sui-theme/lib/index';
+
+.sui-MoleculeTabs {
+  display: inline-flex;
+  &-scroller {
+    @include reset-list;
+    display: inline-flex;
+    flex-direction: row;
+    flex-wrap: nowrap;
+    justify-content: flex-start;
+    align-items: flex-start;
```

Comment:
please sort the props

please run `npm run lint` and fix the molecule tabs warnings